assets/js/editor.js

```diff
@@ -321,6 +322,7 @@ $gp.editor = (
 					success: function( data ) {
 						$gp.notices.success( 'Saved!' );
 						$gp.editor.replace_current( data );
+						$gp.editor.next();
```

Comment:
There were two reasons I added the `next()`, first was that it called the right logic to display things correctly and second was to keep consistent with how other actions in the editor work (any time you it save, cancel, set a status, etc. the editor moves to the next entry).

I can see though in this case that may not be the desired outcome so I've updated the PR and removed some logic that was based on the assumption that whenever `replace_current()` was called that the preview would be visible (the old logic was a little redundant all things considered).